commands/pipelines/setup.js

```diff
@@ -12,6 +12,16 @@ const PIPELINE_MAX_LENGTH = 30 - STAGING_APP_INDICATOR.length
 const ERR_PIPELINE_NAME_LENGTH = `Please choose a pipeline name between 2 and ${PIPELINE_MAX_LENGTH} characters long`
 const ERR_REPO_FORMAT = 'Repository name must be in the format organization/repo'
 
+const DEFAULT_SETTINGS = {
+  'auto_deploy': true,
+  'wait_for_ci': true,
+  'pull_requests': {
+    enabled: true,
```

Comment:
Oh I had in my mind that snake case keys were an error in Standard but apparently they're not. Changed.